static/app/views/performance/newTraceDetails/traceApi/useTraceMeta.tsx

```diff
@@ -63,4 +67,23 @@ export function useTraceMeta(
       enabled: !!trace && !!organization.slug,
     }
   );
+
+  // When projects don't have performance set up, we allow them to view a sample transaction.
```

Comment:
This is near identical comment, you can just create an inline link referencing the comment from above.

Reply:
I would love to just keep the comment here and not have to jump between files to read the whole thing